packs/agents-of-edgewatch-bestiary/agradaemon.json

```
@@ -696,7 +696,7 @@
                 },
                 "category": "offensive",
                 "description": {
-                    "value": "<p>The agradaemon regains 15 Hit Points, and a pledged fanatic of the agradaemon's choice that it can see loses 15 Hit Points (no effects apply that would decrease this Hit Point loss).</p>\n<p>A pledged creature that dies in this way explodes and deals @Damage[9d10[fire]] damage in a @Template[emanation|distance:20] (@Check[reflex|dc:41|basic] save).</p>"
+                    "value": "<p>The agradaemon regains @Damage[15[damage,healing,untyped]] Hit Points, and a pledged fanatic of the agradaemon's choice that it can see loses 15 Hit Points (no effects apply that would decrease this Hit Point loss).</p>\n<p>A pledged creature that dies in this way explodes and deals @Damage[9d10[fire]] damage in a @Template[emanation|distance:20] (@Check[reflex|dc:41|basic] save).</p>"
```

```suggestion
                    "value": "<p>The agradaemon regains @Damage[15[untyped]] Hit Points, and a pledged fanatic of the agradaemon's choice that it can see loses 15 Hit Points (no effects apply that would decrease this Hit Point loss).</p>\n<p>A pledged creature that dies in this way explodes and deals @Damage[9d10[fire]] damage in a @Template[emanation|distance:20] (@Check[reflex|dc:41|basic] save).</p>"
```
This feels a bit janky like that.